src/Composer/ComposerWidget.vala

```diff
@@ -51,19 +53,96 @@ public class Mail.ComposerWidget : Gtk.Grid {
         }
         set {
             _has_recipients = value;
-            update_send_sensitivity ();
+            send.sensitive = has_recipients;
         }
     }
 
-    private void update_send_sensitivity () {
-        send.sensitive = has_recipients;
+    public ComposerWidget (bool show_subject_field = true) {
+        Object (has_subject_field: show_subject_field);
```

Comment:
Since this is currently a construction property, would it be totally overkill to have:

```
public ComposerWidget () {
    Object (has_subject_field: false);
}

public ComposerWidget.with_subect () {
    Object (has_subject_field: true:);
}
```

I feel like that would be a little less "mystery meat" looking from the outside when you're doing `new ComposerWidget (true);` vs `new ComposerWidget.with_subject ()`